web/src/beta/features/Editor/index.tsx

```diff
@@ -132,8 +133,9 @@ const Editor: React.FC<Props> = ({ sceneId, projectId, workspaceId, tab }) => {
     nlsLayers,
     currentPage,
     currentCamera,
-    showSceneSettings: selectedSceneSetting,
+    showSceneSettings: !!selectedSceneSetting,
```

Comment:
This isn't needed anymore. This can be achieved by just checking if selectedSceneSetting exists inside of the left panel

Reply:
done